app/helpers/application_helper.rb

```diff
@@ -1,2 +1,13 @@
 module ApplicationHelper
+  def current_customer
+    Customer.find_by(id: session[:customer_id]) if session[:customer_id].present?
+  end
```

Comment:
No need to define this helper method. You can use the logic from application controller.